<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamTestDialog</class>
 <widget class="QDialog" name="StreamTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(9, 11, 13);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>开始停止按钮</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SwitchButton" name="switch_stream_test" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>103</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>103</width>
           <height>26</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>测试总次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_task_total_count">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>已测次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_task_count">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>异常总次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_errod_count_">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="stream_text_browser">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_stream_clear_log">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(36, 44, 51);</string>
         </property>
         <property name="text">
          <string>Clear Log</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
           </property>
           <property name="text">
            <string>异常信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>color异常次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_color_error_count">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>depth异常次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_depth_error_count">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>phase异常次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_phase_error_count">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>ir异常次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ir_error_count">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>uvc异常次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_uvc_error_count">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">src\weight\switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
